lntest/btcd.go

```diff
@@ -118,15 +120,29 @@ func NewBackend(miner string, netParams *chaincfg.Params) (
 		}
 
 		// After shutting down the chain backend, we'll make a copy of
-		// the log file before deleting the temporary log dir.
-		logFile := baseLogDir + "/" + netParams.Name + "/btcd.log"
-		logDestination := fmt.Sprintf(
-			"%s/output_btcd_chainbackend.log", GetLogDir(),
-		)
-		err := CopyFile(logDestination, logFile)
+		// the log files, including any compressed log files from
+		// logrorate, before deleting the temporary log dir.
+		files, err := ioutil.ReadDir(baseLogDir + "/" + netParams.Name)
```

Comment:
nit: could have `logDir= baseLogDir + "/" + netParams.Name` so that we can reuse this below

Reply:
Nice find. Update on the way. Just running some tests.